### DISCUSSION TITLE
Request for gear colour editor

### DISCUSSION THREAD

**Discussion by yao108** [xivdev/Penumbra#267] [Ideas]
Would like to see a way to edit gear colour live in game. Having to log out and in to edit in TT is too much of a hassle.

**Comment by munkviken** [comment]
It already exists under Advanced Editing > Materials.

**Reply by yao108** [reply]
Oh wow, thank you very much. Had not realized this feature, my mistake. 

Is it only possible to do this with gear contained in a mod pack? In TT we can edit the color right under the texture map of a gear piece. Is there an equivalent method for penumbra?
